poetry_scripts.py

```diff
@@ -1,9 +1,9 @@
 import os
+import shutil
 import sys
 from pathlib import Path
-import shutil
-import pdoc as p
-from pdoc import render
+
+# ruff: noqa: E501
```

Comment:
Ignore line length in this file

Reply:
Why do we need this?

Reply:
Ok, I fixed the string that was too long.